### DISCUSSION TITLE
Multitarget Regression in TimeSeries

### DISCUSSION THREAD

**Discussion by AdamTheDestroyer** [pycaret/pycaret#1597] [Q&A]
Hello together,

is there a possebility to do a multitarget regression in Time Series?

My Case:
3000 feature columns, 1 target column, 40 future time data columns.
Until now:
1. step: I predict for every of the 3000 feature columns the future with the 40 future time data columns.
2. step; I predict my 1 target column with the 3000 generated future features + future time data.

This takes 9 hours.
Is there a possebility to speed this up and to do "automatic" multitarget predictions?
Until now i do a step by step loop for every colomn of the 3000 feature column.

Greetings Adam

**Comment by ngupta23** [comment]
Could you clarify -- What is the difference between the target column and the 40 future data columns?
Maybe providing a dummy example would help.

Thanks!
